src/pages/Block.vue

```diff
@@ -52,6 +52,9 @@ export default {
       transaction-table( :title="block" :filter="{block}" )
 </template>
 <style scoped lang="sass">
```

Comment:
Shall we use global `scss` instead?

```suggestion
<style lang="scss">
```

Reply:
There is a refactor ticket for styles in the application. For this PR i think it is out of scope to remove scoped styles because of how many changes there are already (3k+ lines)

Reply:
and the styles here are written in a way that would be problematic if they were global I think (unless they were refactored to use BEM)